manifests/charts/UPDATING-CHARTS.md

```diff
@@ -59,7 +59,7 @@ These manifests can be found in [gen-istio.yaml](../charts/istio-control/istio-d
 To regenerate the manifests, run:
 
 ```bash
-$ make copy-templates gen-kustomize update-golden
```

Comment:
why was this removed?

Reply:
Yes I think encountered the same thing before